### PR TITLE
LPS-198875 upgrade plexus-utils version 3.0.24

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: modules/util/portal-tools-soy-builder/build.gradle [hunk 1]
```diff
@@ -15,8 +15,8 @@ configurations {
 }
 
 dependencies {
-	api group: "com.google.template", name: "soy", version: "2019-03-07"
```
Comment (drewbrokke) [inline modules/util/portal-tools-soy-builder/build.gradle:18]: Is the update to the `soy` dependency necessary for this update? If not, let's leave it off.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@simonjhy just one issue, please see my inline comment. There is another pull that already updated the soy dependency version here: https://github.com/brianchandotcom/liferay-portal/pull/142393. Let's just update the plexus dependency in this pull.

Thanks!

**Comment by drewbrokke** [comment]
@simonjhy @Seiphon looks like there is overlapping work between this pull and https://github.com/drewbrokke/liferay-portal/pull/1536. Can you two coordinate and re-send? Thanks!

**Comment by Seiphon** [comment]
resent ther pr here https://github.com/drewbrokke/liferay-portal/pull/1541

**Pull request closed without merging**